python/knowledges/batch_upload/delete_duplicate_files.py

```
@@ -0,0 +1,211 @@
+        json.dump(deletion_log, f, indent=2)
+    
+    # Summary
+    print("\n" + "=" * 60)
+    print("Deletion Summary:")
+    print("=" * 60)
+    print(f"✅ Successfully deleted: {len(deleted_files)} files")
+    print(f"❌ Failed deletions: {len(failed_deletions)} files")
+    print(f"📄 Deletion log saved to: {log_filename}")
+    
+    if failed_deletions:
+        print(f"\nFiles that couldn't be deleted:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)